specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2025-06-01/mfe.json

```diff
@@ -13110,6 +13318,142 @@
       },
       "additionalProperties": false
     },
+    "CapabilityHost": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceBase"
+        }
+      ],
+      "properties": {
+        "aiServicesConnections": {
+          "description": "List of AI services connections.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-nullable": true
+        },
+        "capabilityHostKind": {
+          "description": "Kind of this capability host.",
+          "default": "Agents",
+          "$ref": "#/definitions/CapabilityHostKind"
+        },
+        "customerSubnet": {
+          "description": "Customer subnet info to help set up this capability host.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "provisioningState": {
+          "description": "Provisioning state for the CapabilityHost.",
+          "$ref": "#/definitions/CapabilityHostProvisioningState",
+          "readOnly": true,
+          "x-ms-mutability": [
```

Comment:
>List of Storage connections. [](http://example.com/codeflow?start=26&length=28)

Similar question here: what are these? From the name, they could be Urls, connection strings, ResourceIds, maybe even resource names, or something else. Add to the description (and maybe the name) to make it less ambiguous.